### PR TITLE
Add examples to README

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +122,11 @@ Use the `Manage Apps` button and add your app using a link:
 
 [safeaddapp]: https://raw.githubusercontent.com/gnosis/safe-apps-sdk/master/assets/third-pary-app-modal.png 'Safe Multisig: Add Safe App'
 
+## Examples of applications built with this SDK
+- https://github.com/rmeissner/sapp-tx-builder
```
Comment (nicosampler) [inline README.md:126]: we will move tx-builder to safe-react-apps, so perhaps we should remove this ref.

**Review by nicosampler — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Uxio0) [reply README.md:126]: Hi @nicosampler, it that is meant to be done soon ok, but it it takes 1 or 2 weeks I would leave it as it's very useful for people wanting to hack into the Safe, and then will remove it when the application is moved :)

Comment (nicosampler) [reply README.md:126]: ok, let's leave it. I'll remove it once we have migrated it.

**Comment by nicosampler** [comment]
@Uxio0  can you create the PR from dev to master, so we can release it. Thx.

**Pull request merged**